internal/generate/tags/README.md

```diff
@@ -24,6 +24,7 @@ Some flags control generation a certain section of code, such as whether the gen
 | --- | --- | --- | --- |
 | `GetTag` |  | Whether to generate GetTag | `-GetTag` |
 | `ListTags` |  | Whether to generate ListTags | `-ListTags` |
+| `ServicePackage` |  | AWS Go SDK package to use. Defaults to the provider service package name. | `-ServicePackage=chimesdkvoice` |
```

Comment:
```suggestion
| `TagServicePackage` |  | AWS Go SDK package to use. Defaults to the provider service package name. | `-TagServicePackage=chimesdkvoice` |
```

Maybe? Seems like this will go away eventually and might be easier to extricate with this name.

Reply:
Or `AWSSDKServicePackage` to match `AWSSDKVersion`?

Reply:
Updated to `AWSSDKServicePackage` and changed the logic around so the `InternalServicePackage` field is no longer necessary. If we opt to back this out later it should only requiring removing the flag and reverting a couple function calls to use the `servicePackage` variable again.